MapzenSDK/Marker.swift

```diff
@@ -411,15 +457,23 @@ public class SystemPointMarker : Marker, GenericSystemPointMarker {
    - parameter coordinates: Coordinates to animate the marker to
    - parameter seconds: Duration in seconds of the animation.
    - parameter ease: Easing to use for animation.
+   - returns: True if successful, false if error or marker has not been added to the map
    */
   public func setPointEased(_ coordinates: TGGeoPoint, seconds: Float, easeType ease: TGEaseType) -> Bool {
-    return tgMarker.setPointEased(coordinates, seconds: seconds, easeType: ease)
+    guard let marker = tgMarker else { return false }
+    marker.pointEased(coordinates, seconds: seconds, easeType: ease)
+    do {
+      try marker.getError()
```

Comment:
Hmmm yeah you're right. Probably should just pull this then and go back through a re-add it when we tackle https://github.com/mapzen/ios/issues/248